Tensile/ClientWriter.py

```diff
@@ -483,7 +485,10 @@ def param(key, value):
         for coFile in codeObjectFiles:
             param("code-object", os.path.join(sourceDir,coFile))
 
-        param('results-file', os.path.join(stepBaseDir, "../Data", stepName+"-new.csv"))
+        if globalParameters["NewClient"] == 1:
+          param('results-file', os.path.join(stepBaseDir, "../Data", stepName+"-new.csv"))
```

Comment:
Should this be if || == 2?

Reply:
No.  If we're not running the old client, the new client needs to write the results file that's used by the subsequent performance analysis steps.